javalin/src/main/java/io/javalin/util/ConcurrencyUtil.kt

```diff
@@ -128,34 +128,38 @@ open class NamedThreadFactory(protected val prefix: String) : ThreadFactory {
 }
 
 open class NamedVirtualThreadFactory(prefix: String) : NamedThreadFactory(prefix) {
-    override fun newThread(runnable: Runnable): Thread = ReflectiveVirtualThreadBuilder()
+    override fun newThread(runnable: Runnable): Thread = VirtualThreadBuilder.create()
         .name("$prefix-Virtual-${threadCount.getAndIncrement()}")
         .unstarted(runnable)
 }
 
-open class ReflectiveVirtualThreadBuilder {
+class VirtualThreadBuilder {
```

Comment:
You can convert this class into the object class (singleton) and get rid of the companion object at all. It'll make more sense, also considering that we're not creating a new instance of it.